cax/main.py

```diff
@@ -3,7 +3,7 @@
 import os.path
 import time
 
-from cax.config import mongo_password, set_json, get_task_list, get_config
+from cax.config import mongo_password, set_json, set_database_log, get_task_list, get_config
```

Comment:
This is getting a bit long... may change this to `from cax import config` then `config.mongo_password` later.